src/core/mac/mac_frame.cpp

```diff
@@ -1061,49 +1061,37 @@ const TimeIe *Frame::GetTimeIe(void) const
 #if OPENTHREAD_CONFIG_MULTI_RADIO
 uint16_t Frame::GetMtu(void) const
 {
-    uint16_t mtu;
+    uint16_t mtu = OT_RADIO_FRAME_MAX_SIZE;
```

Comment:
Fix: `error: 'mtu' may be used uninitialized in this function [-Werror=maybe-uninitialized]`

Reply:
I am wondering which toolchain gives this warning/error.

Reply:
See https://github.com/openthread/ot-br-posix/pull/803/checks?check_run_id=2396712279#step:4:2930